test/SlowTests/Server/Documents/ETL/ConnectionStringTests.cs

```diff
@@ -85,12 +85,12 @@ public void CanUpdateConnectionStrings()
 
                 //update url
                 ravenConnectionString.Url = "http://127.0.0.1:8081";
-                store.Admin.Server.Send(new UpdateConnectionStringOperation<RavenConnectionString>(ravenConnectionString, ravenConnectionString.Name, store.Database));
+                store.Admin.Server.Send(new AddConnectionStringOperation<RavenConnectionString>(ravenConnectionString, store.Database));
```

Comment:
Better to call it `PutConnectionStringOperation` instead of `AddConnectionStringOperation`, Add implies insert only, Put implies overwrite if needed.